cranelift/fuzzgen/src/lib.rs

```diff
@@ -38,19 +42,23 @@ pub struct TestCase {
 
 impl fmt::Debug for TestCase {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
-            r#";; Fuzzgen test case
+        writeln!(f, ";; Fuzzgen test case\n")?;
+        writeln!(f, "test interpret")?;
+        writeln!(f, "test run")?;
 
-test interpret
-test run
-set enable_llvm_abi_extensions
-target aarch64
-target s390x
-target x86_64
+        // Print only non default flags
+        let default_flags = Flags::new(settings::builder());
+        for (default, flag) in default_flags.iter().zip(self.flags.iter()) {
+            assert_eq!(default.name, flag.name);
 
-"#
-        )?;
+            if default.value_string() != flag.value_string() {
+                writeln!(f, "set {}={}", flag.name, flag.value_string())?;
+            }
+        }
+
+        writeln!(f, "target aarch64")?;
+        writeln!(f, "target s390x")?;
+        writeln!(f, "target x86_64\n")?;
```

Comment:
Does it make sense to add `riscv64` here too (or are you worried it would find too many issues)?

Actually in the spirit of your enum `.all()` method above -- perhaps we can define a `cranelift_codegen::isa::all()` that returns all `Triple`s compiled into the build, and use that here? (Feel free to drop that idea though if it's more than ~20 lines of complexity or so.)

Reply:
This ended up not being very easy to do, I might try it again later. But either way I've added `riscv64` to the target list.